sandbox.c

```diff
@@ -135,7 +135,6 @@ int create_sandbox()
     ret = mount("tmpfs", "/", "tmpfs", MS_REMOUNT | MS_RDONLY, "size=0k");
```

Comment:
@giuseppe Why do we need this tmpfs, and why does it fail?

Reply:
was it expected to mount tmpfs on `/tmpfs`, not `/`?

Reply:
I think the "tmpfs" here is just ignored, the goal is to remount the entire root as read only.  We didn't manage to find the reason why it fails in the @Rohrschacht use case